content/design-systems-for-developers/react/pt/build.md

```diff
@@ -0,0 +1,238 @@
+commit: e7b6f00
+---
+
+No capítulo 3 vamos configurar as ferramentas essenciais aos sistemas de design, a começar com o Storybook, o explorador de componentes mais popular.
+O objetivo deste guia é mostrar como as equipas profissionais constroem sistemas de design, e como tal iremos focar-nos nos detalhes mais granulares tais como higiene do código, extras do Stoybook que poupam tempo e na estrutura de pastas ou diretórios.
+
+![Onde é que o Storybook se encaixa](/design-systems-for-developers/design-system-framework-storybook.jpg)
+
+## Formatação de código e linting para a higiene
+
+Os sistemas de design são colaborativos, como tal ferramentas que corrigem a sintaxe e uniformizam a formatação de código irão servir para melhorar a qualidade das contribuições. A aplicação de uma consistência de código através de ferramentas é menos dispendioso do que fazer o policiamento de código manualmente, o que se torna um benefício para o autor do sistema de design.
+
+Neste tutorial vamos usar o VSCode, mas a mesma ideia pode ser aplicada a outros editores modernos, tais como Atom, Sublime, ou IntelliJ.
+
+Se adicionarmos o Prettier ao projeto e configurarmos o editor correctamente, deveremos obter uma formatação consistente sem precisar pensar muito:
+
+```bash
+yarn add --dev prettier
+```
+
+Se estiver a usar o Prettier pela primeira vez, poderá ter que o configurar para o seu editor escolhido. No caso do VSCode instale o seguinte extra:
+
+![extra prettier para o VSCode](/design-systems-for-developers/prettier-addon.png)
+
+Active a opção "Format on Save" `editor.formatOnSave` se ainda não o tiver feito.
+Assim que instalar o Prettier, irá reparar que este automaticamente o código sempre que o ficheiro for guardado.
+
+## Instalar o Storybook
+
+O Storybook é o [explorador de componentes](https://blog.hichroma.com/the-crucial-tool-for-modern-frontend-engineers-fb849b06187a) standard da indústria para construir componentes de IU em isolamento. Visto que os sistemas de design focam-se nestes, o Storybook é a ferramenta ideal para o caso de uso. Iremos aproveitar as seguintes funcionalidades:
+
+- 📕Catálogo de componentes de IU
+- 📄Guardar as variações dos componentes como estórias
+- ⚡️Uso de ferramentas de experiência para o programador tal como o Hot Module Reloading
+- 🛠Suporte de inúmeras camadas de visualização, incluíndo o React
+
+Instalar e executar o Storybook
+
+```bash
+npx -p @storybook/cli sb init
+yarn storybook
+```
+Deverá ver o seguinte:
+
+![Interface de utilizador inicial Storybook](/design-systems-for-developers/storybook-initial.png)
+
+Fantástico, acabámos de configurar o explorador de componentes!
+
+Por defeito, o Storybook, cria uma pasta `src/stories` com alguns exemplos de estórias. No entanto, quando copiámos os nossos componentes, foram copiadas também as suas estórias. Podemos indexá-las no nosso Storybook através da alteração 
```

Comment:
@joaogarin, no, it's actually pt-pt. It's revised to the new Portuguese Orthographic Agreement](https://en.wikipedia.org/wiki/Portuguese_Language_Orthographic_Agreement_of_1990). And my read on the definition of Storybook's use of story/stories, is that you implement some sort of narrative of what the component does/should do. So that's why the word [estória](https://dicionario.priberam.org/est%C3%B3ria) was used. When i was translating this i had a bit of a hard time because mechanically i would write it as "história(s)" due to the fact that it was the word used through out my formative years. I know that the use of this word has it's quirks about it. Some accept it as the correct definition, others don't. I did a bit of extra digging and that's why i went with it. But i can switch it to "história", i have no issues with that.